### DISCUSSION TITLE
Ability to change suggested import name?

### DISCUSSION THREAD

**Discussion by phatsk** [vuejs/language-tools#1208] [Q&A]
Our project has several components like this:

```
- atoms/
  - base-button.vue
  - base-toggle.vue
```

When I start typing an import, the autosuggested import from Volar comes out like this:

```js
import baseButtonVue from './atoms/base-button.vue`
```

Our standards however call for imports like this:

```js
import BaseButton from './atoms/base-button'
```

I don't mind having the `.vue` in the file path, that's not really going to affect our code. However, changing the import is a simple yet repetitive task that I think I should be able to configure either in Volar or through some kind of documentation hint in the component itself, but I can't find any documentation that supports this.

**Comment by phatsk** [comment]
I did find one way that should work for now, though I'd like to keep this discussion open to see if there's another way:

```js
const SomeThing = { /* component code here */ }
export default SomeThing
```

Seems to do what I'm looking for.

**Comment by johnsoncodehk** [comment]
This is expected behavior, see #82.

But you can trigger auto import component in template, it will not end with `Vue`.
<img width="792" alt="螢幕截圖 2022-04-20 14 38 53" src="https://user-images.githubusercontent.com/16279759/164166148-8b17cb17-8393-425e-9acd-e25560adaa9b.png">
<img width="394" alt="螢幕截圖 2022-04-20 14 39 05" src="https://user-images.githubusercontent.com/16279759/164166168-0c4903c9-2ffc-4c10-ac35-8fc08956f265.png">

**Comment by segevfiner** [comment]
It would still be nice to be able to have an option to not add this suffix even when completing in the `script`... I think it might require changes to the tsserver plugin and/or takeover mode to achieve that?

**Reply by johnsoncodehk** [reply]
Please track #82
